tests/Aspire.Hosting.Tests/MySql/AddMySqlTests.cs

```diff
@@ -1,13 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Hosting.MySql;
+using Aspire.Hosting.Utils;
+using Aspire.Hosting.Tests.Utils;
 using System.Net.Sockets;
 using System.Text.RegularExpressions;
```

Comment:
```suggestion
using System.Net.Sockets;
using System.Text.RegularExpressions;
using Aspire.Hosting.MySql;
using Aspire.Hosting.Tests.Utils;
using Aspire.Hosting.Utils;
```

Others should be alphabetical.